syntax = "proto3";

package ecclesia;

message QueryVariables {
  message VariableValue {
    string name = 1;
    // The possible values the variable can take on. These will be ORed together
    // when being used in the redpath predicate.
    repeated string values = 3;
    reserved 2;
  }
  repeated VariableValue variable_values = 2;
  reserved 1;
}
